app/src/main/java/org/kiwix/kiwixmobile/intro/IntroActivity.kt

```diff
@@ -0,0 +1,142 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2019 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+package org.kiwix.kiwixmobile.intro
+
+import android.os.Bundle
+import android.os.Handler
+import android.util.Log
+import android.widget.ImageView
+import androidx.annotation.Nullable
+import androidx.viewpager.widget.ViewPager
+import androidx.viewpager.widget.ViewPager.OnPageChangeListener
+import butterknife.BindView
+import butterknife.OnClick
+import com.pixelcan.inkpageindicator.InkPageIndicator
+import org.kiwix.kiwixmobile.R.id
+import org.kiwix.kiwixmobile.R.layout
+import org.kiwix.kiwixmobile.core.Intents.internal
+import org.kiwix.kiwixmobile.core.base.BaseActivity
+import org.kiwix.kiwixmobile.core.di.components.CoreComponent
+import org.kiwix.kiwixmobile.core.main.CoreMainActivity
+import org.kiwix.kiwixmobile.intro.IntroContract.Presenter
+import org.kiwix.kiwixmobile.intro.IntroContract.View
+import org.kiwix.kiwixmobile.kiwixActivityComponent
+import java.util.Timer
+import java.util.TimerTask
+
+const val DURATION_WAITER = 800
+const val PERIOD_TIME = 2000
+const val DELAY_TIME = 0
+
+class IntroActivity : BaseActivity(), View {
+  private val handler = Handler()
+  private val timer = Timer()
+  @BindView(id.view_pager)
+  @Nullable
```

Comment:
I think we don't need `@Nullable` in kotlin.

Reply:
i used @nullable because there is Binding Problem 
You can refer this  [link](https://github.com/JakeWharton/butterknife/issues/960)

Reply:
We should not use Butterknife ViewBinding in kotlin files. If possible we should use synthetic extension functions https://kotlinlang.org/docs/tutorials/android-plugin.html